File: Year_01/Programming_Paradigms/Labs/pp_lab_04/java/src/Main.java
void main() throws FileNotFoundException {
    List<Person> people = new ArrayList<>();
    
    people.add(new Person("Alexandru-Mihail Ionita", LocalDate.of(2002, 1, 10), "Male", 195));
    people.add(new Person("Alexandru-Mihail Ionita", LocalDate.of(2002, 1, 11), "Female", 180));
    people.add(new Person("Alexandru-Mihail Ionita", LocalDate.of(2002, 1, 11), "Female", 195));
    people.add(new Person("Alexandru-Mihail Ionita", LocalDate.of(2002, 1, 11), "Male", 195));
    people.add(new Person("Diana-Alexia Bordea", LocalDate.of(2004, 2, 20), "Female", 171));
    people.add(new Person("Tiberiu-Adrian Petre",  LocalDate.of(2002, 9, 21), "Male", 178));
    people.add(new Person("Mircea-Dan Deaconu",  LocalDate.of(2003, 02, 20), "Male", 150));
    
    people = people.stream().sorted().toList();

    for(Person person : people)
        System.out.println(person.toString() + "\n");

    people.get(0).store("people.txt");
}

File: Year_01/Programming_Paradigms/Labs/pp_lab_04/python/main.py
from datetime import date
from Person import Person

def main():
    people = [
        Person("Alexandru-Mihail Ionita", date(2002, 1, 10), "Male", 195),
        Person("Diana-Alexia Bordea", date(2004, 3, 28), "Female", 171),
        Person("Tiberiu Petre Adrian", date(2002, 2, 1), "Male", 184),
        Person("Mircea-Dan Deaconu", date(2002, 2, 20), "Male", 150)
    ]

    people = sorted(people)

    for person in people:
        print(person)
    
    people[0].store("people.txt")

if __name__ == "__main__":
    main()
